apps/practices/src/components/practice-details/PracticeSectionDetails.tsx

```diff
@@ -30,9 +67,42 @@ export const PracticeSectionDetails = ({
 
   return (
     <CardInfo
-      icon={<Icon fontSize="medium" name="error" />}
       items={filteredMetadataListItems}
       title={t("practiceDetails.info.title")}
     />
   );
 };
+
+export const PracticeTypeDetails = ({
+  tipoPratica,
+}: PracticeSectionTypeDetailsProps) => {
```

Comment:
What do you think if we move this component to another file under the `practice-details` folder?